plugin/storage/es/spanstore/service_operation.go

```diff
@@ -16,6 +16,7 @@ package spanstore
 
 import (
 	"context"
+	"crypto/sha256"
```

Comment:
there's no need for crypto hash, see https://github.com/jaegertracing/jaeger/blob/d16de316a8bfc0ba7a6d5d1b7e43d3c48156d8ae/model/hash.go#L30-L36